gridfs/asynchronous/grid_file.py

```diff
@@ -1842,6 +1845,9 @@ async def next(self) -> bytes:
 
     __anext__ = next
 
+    async def to_list(self) -> list[bytes]:
```

Comment:
AsyncGridOutIterator/GridOutIterator are never used so I vote we remove to_list. I'm not sure why GridOutIterator even exists, maybe for backwards compat? We don't even have tests for it.

Reply:
Three tests fail if this one is removed:

```
FAILED test/asynchronous/test_session.py::TestSession::test_gridfs - AttributeError: 'dict' object has no attribute 'read'
FAILED test/asynchronous/test_session.py::TestSession::test_gridfs_bucket - AttributeError: 'dict' object has no attribute 'read'
FAILED test/asynchronous/test_session.py::TestSession::test_gridfsbucket_cursor - AttributeError: 'dict' object has no attribute 'read'
```

Reply:
Whoops sorry wrong one, this change is fine.

Reply:
Are you sure? AsyncGridOutIterator is never used anywhere so I don't see how that's possible.

Reply:
Yeah I read the wrong line numbers for this change, removed the new `to_list` methods that aren't for `AsyncGridOutCursor`.